src/server-extension/query/tokenEntityByOwner.ts

```diff
@@ -0,0 +1,48 @@
+export const tokenEntityByOwner = `WITH CheapestNFT AS (
+    SELECT 
+        ne.token_id,
+        MIN(ne.price) AS "cheapestNFTPrice"
+    FROM 
+        nft_entity ne
+    WHERE 
+        ne.current_owner = $1
+    GROUP BY
+        ne.token_id
+)
+
+SELECT
+    t.id as id,
+    t.name as name,
+    t.image as image,
+    t.media as media,
+    t.metadata as metadata,
+    me.id as "metaId",
+    me.description as "metaDescription",
+    me.image as "metaImage",
+    me.animation_url as "metaAnimationUrl",
+    t.block_number as "blockNumber",
+    t.created_at AS "createdAt",
+    t.updated_at AS "updatedAt",
+    COUNT(ne.id) as count,
+    
+    c."cheapestNFTPrice",
+    
+    col.id AS "collectionId",
+    col.name AS "collectionName"
+FROM
+    token_entity as t
+    LEFT JOIN nft_entity as ne ON t.id = ne.token_id AND ne.current_owner = $1
+    LEFT JOIN CheapestNFT as c ON t.id = c.token_id
+    LEFT JOIN collection_entity as col ON t.collection_id = col.id
+    LEFT JOIN metadata_entity as me ON t.meta_id = me.id
```

Comment:
4 left joins is not a performance wise

Reply:
I have benchmarked a few queries, some with 3 and some with 4 joins

tested by a bash script that runs an SQL query 200 times against the fully loaded DB on localhost

results are, with no significant difference between the approaches

mean between 1.55 to 1.65 ms
standard deviation  ~0.25 ms